python/pipdeptree.py

```diff
@@ -0,0 +1,636 @@
+from __future__ import print_function
```

Comment:
Question - did you see that the script generated by the *addresources.go* script, runs properly after it is created?

Reply:
Yes.